controller/config.go

```diff
@@ -271,10 +271,24 @@ const (
 	// Can be set to "legacy", "snapstore", "local" or "local-dangerous".
 	// Cannot be changed.
 	JujudControllerSnapSource = "jujud-controller-snap-source"
+
+	// SSHServerPort is the port used for the embedded SSH server.
+	SSHServerPort = "ssh-server-port"
+
+	// SSHMaxConcurrentConnections is the maximum number of concurrent SSH
+	// connections to the controller.
+	SSHMaxConcurrentConnections = "ssh-max-concurrent-connections"
```

Comment:
`ssh-server-connection-limit` would be a better name.

Reply:
Wasn't sure how I felt on this, so I asked AI :laughing: 
![image](https://github.com/user-attachments/assets/d1c88db8-671f-464c-9c7e-b31a0d6b4cdc)

I'll go with AI on this as I agree with the points made